doc/sphinx/proof-engine/vernacular-commands.rst

```diff
@@ -967,14 +967,17 @@ Controlling display
 
 .. opt:: Warnings "{+, {? {| - | + } } @ident }"
 
-   This :term:`option` configures the display of warnings. It is experimental, and
+   This :term:`option` configures the display of warnings. It
    expects, between quotes, a comma-separated list of warning names or
    categories. Adding - in front of a warning or category disables it, adding +
-   makes it an error. It is possible to use the special categories all and
-   default, the latter containing the warnings enabled by default. The flags are
+   makes it an error. It is possible to use the special categories `all` and
+   `default`, the latter containing the warnings enabled by default. The flags are
    interpreted from left to right, so in case of an overlap, the flags on the
```

Comment:
Also, the doc should mention that, IIUC, in the warning below `unsupported-attributes` is the warning name and `parsing` is the warning category.  I'm not aware that this detail is covered anywhere.

`This command does not support this attribute: warnings. [unsupported-attributes,parsing]`